firewood/src/manager.rs

```diff
@@ -167,7 +167,7 @@ impl RevisionManager {
     ///    The proposal's parent must be the last committed revision, otherwise the commit fails.
     ///    It only contains the address of the nodes that are deleted, which should be very small.
     /// 2. Revision reaping. If more than the maximum number of revisions are kept in memory, the
-    ///    oldest revision is reaped.
+    ///    oldest revision is reaped if not referenced by `RootStore`.
```

Comment:
This comment is confusing to me. It's not clear by what "referenced by `RootStore`" means.

What I'm reading this as is "if RootStore knows about this revision, then don't reap it" which isn't what you want. Reaping here refers to removing the revision from memory, which we want.

I'm also worried about when we don't have a RootStore. Aren't all revisions "not referenced by `RootStore`" in that case? Which implies we're never reaping any revisions, which clearly is wrong.